app/sass/components/_pages-search.scss

```diff
@@ -140,12 +206,20 @@ $ancho:$unidad * 2;
   content:"\e615";
       content: "\e626";
 }
+.indicatorIcon a:before {
+  content: "\e80a";
+}
+.datasetIcon a:before {
+  content: "\e611";
+}
+
 .browse-filters-block > ul {
-  max-height: 200px;
```

Comment:
@mmilaprat @bullz This is needed because the facets can be a long list so i added scrolling and also the cursor needs to be changed to pointer as facets are clickable